### PR TITLE
Add Hyland as partner

### PR DESCRIPTION
**Pull request by aborroy**
### Description
Add Hyland as partner

### CODE REVIEW

**Comment by dtaivpp** [comment]
Hey @aborroy, happy to add this but we first need commits signed with a DCO. Here are a [few tips on retroactively applying a sign off.](https://github.com/src-d/guide/blob/9171d013c648236c39faabcad8598be3c0cf8f56/developer-community/fix-DCO.md) Thanks!

**Comment by aborroy** [comment]
Fixed! Thanks for the hint.

File: _partners/hyland.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ name: Hyland
+ logo: '/assets/media/partners/hyland.png'
+ link: 'https://hyland.com'
+ ---
```
Comment (krisfreedain) [inline _partners/hyland.md:5]: @aborroy 
Please remove the extra single space at the beginning of lines 2-5 
(Jekyll isn't reading it correctly with that tiny issue) 
thanks

**Review by krisfreedain — CHANGES_REQUESTED** [review]
small change needed

**Comment by krisfreedain** [comment]
Much better @aborroy - thank you. A couple minor fixes now. 
The image seem to line up better if they're square - see example: 
Screen Shot 2022-09-26 at 9 51 36 AM

Lastly, please make sure you have DCO sign off on the PR as well - once these two things are done we should be good to go

**Comment by aborroy** [comment]
Since this one has become a bit messy, I've created a new PR in #1023

**Pull request closed without merging**